src/common/variables.c

```diff
@@ -139,7 +140,7 @@ static void init_expansion(dt_variables_params_t *params, gboolean iterate)
 
     params->data->flags = img->flags;
 
-    dt_image_cache_read_release(darktable.image_cache, img);
+    if(params->img == NULL) dt_image_cache_read_release(darktable.image_cache, img);
```

Comment:
And when is params->img released if set ?

Reply:
My mistake ... :cry:

Reply:
Sorry, thinking better. For me that's correct. 
params->img set, means that the cache is already got, in that case in _import_image_internal().
So, for this time (dt_exif_apply_default_metadata()), init_expansion, doesn't need not to get it, neither to release it.

Reply:
I see now, thanks.